syntax = "proto3";
package traces;
import "google/protobuf/struct.proto";

service DatalakeManagerService {
    rpc InsertTraceData (InsertTraceRequest) returns (InsertTraceResponse);
  }
  
  message InsertTraceRequest {
    string path = 1;
    google.protobuf.Struct data = 2;
  }
  
  message InsertTraceResponse {
    string status = 1;
  }
